### PR TITLE
Introduce service task stats

### PR DESCRIPTION
**Pull request by orlandohohmeier**
Introduce service task stats - this is needed to implement the service debug tab.

/cc @mlunoe

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @Poltergeist

**Comment by pierlo-upitup** [comment]
LGTM 💯 minus the conflicts!

File: src/js/structs/TaskStats.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import Item from './Item';
+import TaskStat from './TaskStat';
+
+class TaskStats extends Item {
+
+  /**
+   * @return {TaskStat} task ststatistic about all tasks that run with the same
```
Comment (philipnrmn) [inline src/js/structs/TaskStats.js]: typo in `statistics`

File: src/js/structs/__tests__/Service-test.js [hunk 2]
```diff
@@ -494,6 +495,16 @@ describe('Service', function () {
 
   });
 
+  describe('#getTaskStats', function () {
+
+    it('returns task stats instance', function () {
+      let service = new Service({taskStats:{}});
+
+      expect(service.getTaskStats() instanceof TaskStats).toBeTruthy();
```
Comment (philipnrmn) [inline src/js/structs/__tests__/Service-test.js]: No need to update this, but you can also `jasmine.any` to test type. 
`expect(service.getTaskStats()).toEqual(jasmine.any(TaskStats))`

Comment (orlandohohmeier) [inline src/js/structs/__tests__/Service-test.js]: Didn't know that. That's cool!

**Comment by philipnrmn** [comment]
Awesome work :) especially liking the jsdoc on `TaskStats`.

**Comment by Poltergeist** [comment]
Awesome work as mentioned before awesome work on the JSdoc thanks for going the extra mile on that one! LGTM feel free to merge if you are ready.

**Pull request merged**